docs/project-backlog.md

```diff
@@ -0,0 +1,122 @@
+
+* **OP**: Original Poster. This is the person who has opened or posted the issue.
+
+* **Contributor**: Person(s) that are actually doing the work related to the ticket. Once work is done, they work with the Reviewer and the Contributing Reviewers to get feedback implemented and complete the work. They are responsible for making sure issue status label is up to date.
+
+* **Reviewer**: Person(s) whose Approval is needed to merge the PR.
+
+* **Contributing Reviewer**: Person who provides feedback on an PR but whose Approval is not necessarily needed to merge the PR.
+
+## SLAs
+We value having a responsive backlog, and have response SLAs for each role. If an issue exceeds an SLA, it is put into the `stale` state.
+
+* **Triager**: Daily (Business Days) review of new issues with initial feedback to OP or issue processed within 3 days of last activity.
+* **Reviewer / Contributing Reviewer**: Provide feedback within 3 days of the last activity on a PR
+* **Contributor**: 3 day updates as comments on the PR to indicate activity
+
+## Issue States
+
+### New
+label: `new`
+
+New issues are the domain of the backlog Triagers. Every new issue should be responded to quickly, with a greeting and any needed clarifying questions. If assessing the issue requires more information, the Triager collaborates with the OP until workability is determined.
+
+The most common next steps for new tickets:
+* `available` for actionable work items.
+* `needs-discussion` for issues which cannot be triaged quickly, but raise important points.
+* `no-further-action` for issues which can be quickly resolved.
+
+### Available
+label: `available`
+
+For new work requests Once it has been classified, a ticket is up for grabs to be picked up to be worked on by a contributor.
+
+All work is labeled with a priority:
+* `P0` = Critical (Drop everything and work on it.)
+* `P1` = Important (Work on it if no P0 are present. Use judgment to prioritize items within a list of P1s)
+* `P2` = Low (Work on it when no P1 are left or some one from community wants to contribute)
+
+Work can also be classified into a number of categories, to make available issues easier to find. Common examples of work categories are:
+* `feature-request`
+* `enhancement`
+* `bug`
+* `docs`
+
+Available work can be assigned in the following ways:
+* A contributor assigns the ticket to themselves.
+* A maintainer assigns the ticket to a contributor.
+
+All tickets transition to `wip` once they have been assigned.
+
+### WIP
+label: `wip`
+
+Any ticket which has been assigned a lead contributor is considered work in progress. 
```

Comment:
do you envision some automation?

Reply:
Also - non-members can declare that they are working on issue by mentioning themselves in comments. It is mentioned in CONTRIBUTING.md